doc/api/http.md

```diff
@@ -744,6 +744,27 @@ added: v0.4.0
 
 * {string} The request path.
 
+### `request.method`
+<!-- YAML
+added: v0.4.0
+-->
+
+* {string} The request method.
+
+### `request.host`
+<!-- YAML
+added: v0.4.0
```

Comment:
If we're just now adding these properties, then this version is wrong both here and for `protocol`. So I think it should be `REPLACEME` instead, which should get automatically replaced during the next release.

Reply:
I look the example,  code format like below it. however for the added version, what should it ?
```
### `request.method`
<!-- YAML
added: ?
changes:
  - version: REPLACEME
    pr-url: https://github.com/nodejs/node/pull/33803
    description: Add `method` return 
-->
```

Reply:
@mscdex thanks, I got it